spring-boot-ecommerce/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.datasource.url=jdbc:mysql://localhost:3306/full-stack-ecommerce?useSSL=false&useUnicode=yes&characterEncoding=UTF-8&allowPublicKeyRetrieval=true&serverTimezone=UTC
-spring.datasource.username=ecommerceapp
-spring.datasource.password=ecommerceapp
+spring.datasource.username=root
```

Comment:
Dont push you database specific properties.

Reply:
Revert back this file

Reply:
Ok